timetracker/vms/models.py

```diff
@@ -747,6 +748,39 @@ def is_approved(self):
         """
         return hasattr(self, 'approval')
 
+    @property
+    def total_time(self):
+        """
+        Get the total time from this time record.
+
+        Returns:
+            The total time in this time record
+            that the employee has worked in seconds,
+            rounded to the nearest 15 minute increment.
+        """
+        hours = datetime.timedelta(0)
+        if self.time_end:
+            hours += self.time_end - self.time_start
+            seconds_worked = hours.total_seconds()
+            total_hours = seconds_worked / (60 * 60)
+            return math.floor(total_hours * 2) / 2
+        else:
+            return None
```

Comment:
You can (and should) use the functions from `vms.time_utils` to round time. Additionally, it doesn't make sense to me to have a `total_time` property return `None`. I would return `0` and let the caller handle checking if the time record is complete.